### ISSUE TITLE
The dense ResNet-50 Acc. is unclear to me. Is it 77.1%?

### ISSUE THREAD

**Issue by Ikarosy** [Yanqi-Chen/LATS#1]
Hi, I wonder what is the baseline accuracy of the dense ResNet-50 on ImageNet.  The paper is written densely, but the baseline performances of dense models are neither denoted in figure 2 nor table 3. I can only find the following sentence:"Astonishingly, we observe that even though the performance of the dense network slightly degrades from 77.1 to 76.78, the accuracy of the sparse ones is improved overall."

So, I speculate the top-1 acc. of the dense ResNet-50 in this work is 77.1. It seems like the baseline acc. is a little bit higher than the normal case where 76.13 or 76.15 is often reported.  Will it be possible for you to report the acc. drop just like what you did in your previous works?

And, what about the dense baselines of other works? Do they have the same performance as you did in this work? 

Forgive me for my ignorance, and thanks for your reply in advance. I am just curious. Let it be assured by me that I only want to ask the questions purely out of curiosity and without any offensive intention, after I followed your series of solid works.

**Comment by Yanqi-Chen** [comment]
> So, I speculate the top-1 acc. of the dense ResNet-50 in this work is 77.1.

Correct. In practice, the Top-1 acc. of dense networks fluctuates between 77%~77.1%. To avoid controversy like underestimated dense performance, we finally decide to report a high bar.

> It seems like the baseline acc. is a little bit higher than the normal case where 76.13 or 76.15 is often reported.

When you say normal, I suppose you mean the original training setting in Kaiming's work. In fact, there have been a lot of training tricks developed since then, including cosine annealing learning rate, warmup epochs, etc.  To clarify, we adopt these tricks for a fair comparison with recent studies like STR, Woodfisher, and Powerpropagation. They should not be interpreted as (and are definitely not) part of the pruning algorithm.

> Will it be possible for you to report the acc. drop just like what you did in your previous works

The accuracy is presented in Appendix I numerically. With dense accuracy of 77.1, acc. drop should be accessible to all.

> And what about the dense baselines of other works? Do they have the same performance as you did in this work?

In our experience, a study using the same setting (batch size of 128, 100 training epochs, cosine annealing, warmup, etc.) should always give the same Top-1 acc. around 77%. You will find similar results in STR and WoodFisher (77.01%). In the meantime, some of which with a larger batch size will observe a slightly degraded dense performance of ~76.8% (Powerpropagation, RigL) as we state in the text (degrade to 76.78).

> Forgive me for my ignorance, and thanks for your reply in advance. I am just curious. Let it be assured by me that I only want to ask the questions purely out of curiosity and without any offensive intention, after I followed your series of solid works.

Never mind. These details are indeed necessary for fair comparison within the field of pruning. Feel free to contact me if you have any further questions.

**Comment by Ikarosy** [comment]
Thanks for your reply and your contribution to soft pruning.  Following your consecutive works has made the survey easier for me. And, your patience is appreciated.
